<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editdirector</class>
 <widget class="QDialog" name="editdirector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="key">
   <property name="geometry">
    <rect>
     <x>55</x>
     <y>50</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Director table number</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Edit</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="fullname">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>90</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="school">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="salary">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="table_num">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>50</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>160</y>
     <width>41</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>M</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="radioButton_2">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>160</y>
     <width>31</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>F</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="experience">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>230</y>
     <width>113</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>66</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Full name</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>School</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>51</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Gender</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>190</y>
     <width>41</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Salary</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>125</x>
     <y>230</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Experience</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
